package.json

```diff
@@ -33,14 +33,15 @@
     "eslint-plugin-n": "^15.6.0",
     "eslint-plugin-promise": "^6.1.1",
     "husky": "^9.1.6",
+    "jsonwebtoken": "^9.0.2",
     "semantic-release": "^24.1.3",
     "tsup": "^8.3.0",
     "typescript": "^5.6.3",
-    "vitest": "^2.1.4",
-    "jsonwebtoken": "^9.0.2"
+    "vitest": "^2.1.4"
   },
   "dependencies": {
-    "@types/node": "^18.11.11"
+    "@types/node": "^18.11.11",
+    "luxon": "^3.5.0"
```

Comment:
Please remove luxon dep

The SDK needs to remain mostly free of external dependencies. 
This is important to keep the SDK un opinionated.

This means the SDK can be loaded anywhere (browser or node) and the users of the SDK aren't forced to install additional dependencies into their project just for an SDK

Reply:
removed.